lib/build.gradle

```diff
@@ -36,6 +36,10 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            // https://github.com/robolectric/robolectric/issues/5115
+            all {
+                systemProperty("javax.net.ssl.trustStoreType", "JKS")
```

Comment:
Required to be set before `MockServer` can be instantiated. See the referenced issue for details.